pyensembl/biotypes.py

```diff
@@ -244,6 +244,8 @@
     'IG_J_pseudogene',
     'IG_pseudogene',
     'IG_V_pseudogene',
+    # Found in ftp://ftp.ensembl.org/pub/release-81/gtf/mus_musculus
+    'IG_D_pseudogene',
```

Comment:
Any ideas for automatically generating this biotypes list? I'm terrified of how perpetually out of date this list will always be.